config/comms.txt

```diff
@@ -1,10 +1,22 @@
 ## Communication key for receiving data through world/Topic(), you don't want to give this out
 #COMMS_KEY defaultkey
 
+## Comms key for more widespread sharing.
+## Can only send and recieve Topic calls for the cross-server communications console function, and has a rate limit as set below
+## You want to give this with server ops you trust
+#COMMS_KEY_INSECURE defaultinsecurekey
+
+## Minimum time between Topics recieved with the insecure key, in deciseconds. Default is 30 seconds
+INSECURE_TOPIC_COOLDOWN 300
+
 ## World address and port for server receiving cross server messages
 ## Use '+' to denote spaces in ServerName
 ## Repeat this entry to add more servers
-#CROSS_SERVER ServerName byond:\\address:port
+#CROSS_SERVER ServerName byond://address:port
+
+## World address and port for servers we can contact using the insecure comms key
```

Comment:
You should mention that a server shouldn't be listed in both secure and insecure. Going off of your comms console code that'd cause message dupes.